src/main/java/frc/trigon/robot/subsystems/swerve/SwerveConstants.java

```diff
@@ -83,12 +79,15 @@ public class SwerveConstants {
                     new PIDConstants(4, 0, 0) :
                     new PIDConstants(3, 0, 0);
     private static final double
-            MAX_ROTATION_VELOCITY = RobotHardwareStats.isSimulation() ? 720 : 720,
-            MAX_ROTATION_ACCELERATION = RobotHardwareStats.isSimulation() ? 720 : 720;
+            MAXIMUM_ROTATION_VELOCITY = RobotHardwareStats.isSimulation() ? 720 : 720,
+            MAXIMUM_ROTATION_ACCELERATION = RobotHardwareStats.isSimulation() ? 720 : 720;
     private static final TrapezoidProfile.Constraints ROTATION_CONSTRAINTS = new TrapezoidProfile.Constraints(
-            MAX_ROTATION_VELOCITY,
-            MAX_ROTATION_ACCELERATION
+            MAXIMUM_ROTATION_VELOCITY,
+            MAXIMUM_ROTATION_ACCELERATION
     );
+    static final double
+            MAXIMUM_PID_ANGLE = 180,
+            MINIMUM_PID_ANGLE = -180;
```

Comment:
Where is this used?

Reply:
In the swerve constructor, for the rotation pid controller.